src/components/Menu.vue

```diff
@@ -1,5 +1,18 @@
 <template>
   <div id="controls">
+    <span
+      class="nomlogSummary"
+      v-show="session.voteHistory.length"
```

Comment:
It should probably also only be visible when the player is actually in a live session, so I'd suggest checking for `session.sessionId` as well.